tutorials/docker-gcplogs-driver/index.md

```diff
@@ -151,6 +151,70 @@ to use the Google Cloud logging driver when it is started by systemd:
     the [Logs Viewer](https://console.cloud.google.com/logs/viewer) as you did
     before.
 
+## Persisting configuration across reboots
+
+On Container-Optimized OS, files in `/etc/` are writable, but [data does not
```

Comment:
Is there more than one? I was referring to https://cloud.google.com/container-optimized-os/. Maybe I should make it a link